src/devices/Arduino/category.txt

```diff
@@ -0,0 +1,6 @@
+i2c
+usb
+gpio
+adc
+protocol
+spi
```

Comment:
ok as is in this PR, I might change this though when regenerating the listing - the reason for this file is mostly so the listing looks coherent, there is so many categories here I might think of different category for this

Reply:
I agree that one common name for such extension boards would be helpful, but I can't currently think of a good name. "GpioExpander" is not fitting, because it's more than just GPIO.

Reply:
I guess this needs cleaning up once we have a matching category for such boards. The FT4222 also has the category USB, although it is conceptionally very similar to this binding.